### PR TITLE
adapting dod in documentation

### LINKED ISSUE #872: DOD Documentation for StorageInput is not clear
"Maximum permissible depth of discharge. 80 % dod is equivalent to a state of charge of 20 %."

We use this value differently in SIMONA :

`private val minEnergy = eStorage * dod.toEach`

We should either adapt the documentation or change the SIMONA implementation.

### PR DESCRIPTION
**Pull request by SimonHuette**
resolves #872

### CODE REVIEW

**Review by sebastian-peter — DISMISSED** [review]
Please also add a line to the changelog here :)

**Comment by sebastian-peter** [comment]
Postponed until we deal with storages in simona

File: docs/readthedocs/models/input/participant/storage.md [hunk 1]
```diff
@@ -60,19 +60,6 @@ Model of an ideal electrical battery energy storage.
...
-
-   * - lifeTime
-     - h
-     - Permissible hours of full use
-
-   * - lifeCycle
-     - --
-     - Permissible amount of full cycles
-
    * - em
```
Comment (jo-bao) [inline docs/readthedocs/models/input/participant/storage.md:63]: em should be moved to "entity model"

**Review by jo-bao — CHANGES_REQUESTED** [review]
Just a few small points

File: docs/readthedocs/models/input/participant/storage.md (hunk 1 above)
Comment (danielfeismann) [reply docs/readthedocs/models/input/participant/storage.md]: Good catch!

**Review by jo-bao — APPROVED** [review]
Now for me it's fine!

**Pull request merged**